projects/cds/src/merchandising/model/strategy-result.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface MerchandisingProduct {
+export interface StrategyProduct {
```

Comment:
Another thing to understand for me only,

what is the difference between merchandisingProduct and StrategyProduct

Reply:
A `StrategyProduct` is the model of the product as it is returned from our strategy service, then for ease of rendering the carousel we want to use the core spartacus `Product` model (as that is what the existing `cx-carousel` accepts as its item model). As we then need the product level metadata the easiest option seemed to be to extend the core `Product` model and add the metadata to it, that extension is the `MerchandisingProduct`